safe/gis/vector/clean_geometry.py

```diff
@@ -5,6 +5,7 @@
 from safe.definitions.processing_steps import clean_geometry_steps
 from safe.gis.sanity_check import check_layer
 from safe.utilities.profiling import profile
+from safe.common.custom_logging import LOGGER
```

Comment:
Ah nice, I should just import from somewhere, not creating new one everytime :D

Reply:
one line instead of one.
Actually, you can do it from any file with `LOGGER` in it already. But maybe better to use this one in `custom_logging`